src/app/documentation/demos/code/code-highlight-snippet.ts

```diff
@@ -6,15 +6,16 @@
 import {Component} from "@angular/core";
 
 const HTML_EXAMPLE = `
+<!-- "&#123;" left curly brace / "&#125;" right curly brace -->
 <pre>
     <code clr-code-highlight="language-css">
-        .some-component {
+        .some-component &#123;
```

Comment:
sorry, i understand the issue now. there are multiple ways to escape `{` `}` but we dont in particular recommend one. for eg: you could also use `ngNonBindable` in some cases. let me check with the team to see if we need to update the code highlight documentation to reflect that. thanks a lot!